tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,
-    py27-pyramid{12,13,14,15,16},
+    py27,py34,py35,py36,py37,py38,pypy,
```

Comment:
I'd prefer not to have `3.8` in the default envlist until there is at least some notional release for it on https://www.python.org/download/pre-releases/ (OK to test the nightly on Travis, but not required on developer machines).

See: https://www.python.org/download/pre-releases/ for available pre-releases.

Reply:
Fixed in bde8720